cli/tests/activate/hook.exp

```diff
@@ -3,23 +3,13 @@
 
 set dir [lindex $argv 0]
 set flox $env(FLOX_BIN)
-
-set timeout 5
-set cmd "$flox activate --dir $dir"
```

Comment:
we don't need to `set cmd`?

Reply:
cmd was used to enhance the error output, but made the flow harder the understand and easy to mess up.